index.html

```diff
@@ -3,27 +3,27 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1" />
-    <title>Portfolio</title>
+    <title>Portfolio-Software developer Portfolio</title>
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;700&family=Poppins:wght@100;200;300;400;500&family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
     <link
       rel="stylesheet"
       href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
       type="text/css"
     />
-    <link href="styles.css" rel="stylesheet" />
+    <!-- <link href="styles.css" rel="stylesheet" /> -->
   </head>
   <body>
     <header class="header header-flex">
       <nav class="nav flex">
         <a href="#" class="logo">Abimbola</a>
-        <a class="none"><i class="fa fa-bars cursor"></i></a>
+        <a href="#" class="none"><i class="fa fa-bars cursor"></i></a>
       </nav>
       <ul class="nav-list block">
-        <li class="ul-list cursor"><a>Portfolio</a></li>
-        <li class="ul-list cursor"><a>About</a></li>
-        <li class="ul-list cursor"><a>Contact</a></li>
-        <li class="ul-list cursor"><a><i class="fa fa-envelope white header-icon"></i></a></li>
+        <li class="ul-list cursor"><a href="#">Portfolio</a></li>
+        <li class="ul-list cursor"><a href="#">About</a></li>
+        <li class="ul-list cursor"><a href="#">Contact</a></li>
+        <li class="ul-list cursor"><a href="#"><i class="fa fa-envelope white header-icon"></i></a></li>
```

Comment:
- [ ] The color for this is hard to get. I invite you to change it 🐦 
![image](https://user-images.githubusercontent.com/20563252/122103276-0ed0db80-cddc-11eb-82e8-98a2c7b9e69f.png)

Reply:
done 👍